desktop-file/welcome.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
```

Comment:
Second desktop file: I understand you need this for autostart, but nothing indicates this here

Reply:
added notes for contributors to explain the desktop files, and added comments inside the desktop files.